catalog/content/core/content-core-api/pom.xml

```diff
@@ -61,7 +61,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.75</minimum>
+                                            <minimum>0</minimum>
```

Comment:
Yes. Since there are no tests this value can be set to any number and the coverage check will still pass. It make more sense to me to set it at 0 since that's the actual branch coverage.